### DISCUSSION TITLE
Evaluation of model on original image spacing and inference mode

### DISCUSSION THREAD

**Discussion by HripsimeS** [Project-MONAI/tutorials#602] [Q&A] [answered]
Hello dear MONAI team,

I need your help to understand what is going wrong with my evaluation processes of 3D Unet model on my dataset. 
For the training I received the best metric 96%, Epoch Average Loss and Val Mean Dice are also looking quite good. 
But when I am trying to evaluate my model on original image spacing and on inference mode, I don't get any significant results.
Just some more information, for the evaluation of model on original images I had to remove spacingSpacingd and CropForegroundd in test_transforms because my y_true shape was smaller than y_pred. But in the end I received only 6% metric. And for inference mode evaluation, after visualization I don't see any predicted mask. Please find my notebook below:

https://github.com/HripsimeS/Computer-Vision/blob/main/3DUnet_CTPelvic1K-Adam.ipynb

Look forward to hearing from you!

Sincerely

**Comment by yiheng-wang-nv** [comment, marked as answer]
Hi @HripsimeS , does this ticket related to https://github.com/Project-MONAI/tutorials/discussions/596? If it is already solved, I think this ticket can be closed as well : )

**Reply by HripsimeS** [reply]
Hello @yiheng-wang-nv. Yes this ticket is solved as well, thank you :)
